.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'Bug'
```

Comment:
would recommend to add the unverified label to an incoming bug which can be removed when one of us acknowledges the bug.

Reply:
@5-mark  ⬆️

Reply:
@5-mark